### ISSUE TITLE
ML Pipeline

### ISSUE THREAD

**Issue by kash-dotcom** [kash-dotcom/VineFind#3]
As a **ML Engineer,** I want to model the data, so I can demonstrate how the ML pipeline can classify and cluster in the Jupyter Notebook.

**Assessment Criteria**
•	Conduct hyperparameter optimization.
•	Assess feature importance.
•	Select the appropriate algorithms.
•	Use techniques to prevent overfitting, such as early stopping and dropout layers.

**Comment by kash-dotcom** [comment]
The project went in a different directions so techniques to prevent overfitting, such as early stopping and dropout layers. were no longer required. The desired outcome for a MVP was achieved without the need to conduct hyperparameter optimization. In future iterations

A hybrid recommendation logic should be developed that integrates the
categorical features alongside the description embeddings. With pre-filtering
features that would constrain its outputs to **sparkling wine** this would
improve its reliability. Nevertheless, in its current form, customers can
venture out of their comfort zone and find wines that are similar to their
tastes, which was one of the business requirements.
